### PR TITLE
Enable UseAutoProperyAnalyzer - this should be fixed with the v2 IDE …

### LINKED ISSUE #5408: Investigate performance impact of UseAutoPropertyAnalyzer...
We're seeing huge regressions in RPS since this was introduced (and possibly in upstream perf testing).  @rchande @CyrusNajmabadi @kevinh-ms need to sync on how we need to test this before re-enabling.

### PR DESCRIPTION
**Pull request by mavasani**
…diagnostic engine.

Fixes #5408

### CODE REVIEW

**Comment by mavasani** [comment]
Tagging @CyrusNajmabadi @KevinH-MS @heejaechang

**Comment by heejaechang** [comment]
:+1:

File: src/EditorFeatures/VisualBasic/UseAutoProperty/UseAutoPropertyAnalyzer.vb [hunk 1]
```diff
@@ -9,8 +9,8 @@ Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
 
 Namespace Microsoft.CodeAnalysis.Editor.VisualBasic.UseAutoProperty
     ' https://github.com/dotnet/roslyn/issues/5408
```
Comment (balajikris) [inline src/EditorFeatures/VisualBasic/UseAutoProperty/UseAutoPropertyAnalyzer.vb]: Remove link to bug now that its fixed?

**Comment by CyrusNajmabadi** [comment]
:+1: Thanks!

**Comment by mavasani** [comment]
Tagging @basoundr - we may want to verify rolling diagnostic perf run numbers before and after this change.

**Comment by basoundr** [comment]
@mavasani I am starting a run now. Once we get the numbers, you can merge this change and we can have another run to get the new numbers

File: src/EditorFeatures/VisualBasic/UseAutoProperty/UseAutoPropertyAnalyzer.vb (hunk 1 above)
Comment (mavasani) [inline src/EditorFeatures/VisualBasic/UseAutoProperty/UseAutoPropertyAnalyzer.vb]: Thanks, removed now.

**Comment by mavasani** [comment]
@basoundr is this good to be merged?

**Comment by basoundr** [comment]
@mavasani The tests are still running for before-changes. It is going to take a while before it is done

**Comment by mavasani** [comment]
Synced with @basoundr - he is fine to merge this and validate with a perf rolling run later.

**Comment by basoundr** [comment]
In general, the disk reads have improved drastically. Both VM and Memory allocation looks good

**Comment by basoundr** [comment]
Note that these results are for single iteration so there is a high probability of noise associated with this result

**Pull request merged**